src/Admin/Views/Shared/_OrganizationFormScripts.cshtml

```diff
@@ -103,12 +94,52 @@
                 document.getElementById('@(nameof(Model.SelfHost))').checked = true;
                 document.getElementById('@(nameof(Model.UseResetPassword))').checked = true;
                 document.getElementById('@(nameof(Model.UseScim))').checked = true;
-                // Licensing
-                document.getElementById('@(nameof(Model.LicenseKey))').value = '@Model.RandomLicenseKey';
-                document.getElementById('@(nameof(Model.ExpirationDate))').value = '@Model.FourteenDayExpirationDate';
-                document.getElementById('@(nameof(Model.SalesAssistedTrialStarted))').value = true;
             break;
         }
-        
     }
-</script>
\ No newline at end of file
+
+    /***
+    * Set Secrets Manager values based on current usage (for migrating from SM beta or reinstating an old subscription)
+    */
+    function setInitialSecretsManagerConfiguration() {
+        // Seats
+        document.getElementById('@(nameof(Model.SmSeats))').value = Math.max(@Model.OccupiedSmSeatsCount, 1);
+
+        // Service accounts
+        const baseServiceAccounts = getBaseServiceAccountValue('@((byte)Model.PlanType)');
+        if ('@((byte)Model.PlanType)' !== '@((byte)PlanType.Free)' && @Model.ServiceAccountsCount > baseServiceAccounts) {
+            document.getElementById('@(nameof(Model.SmServiceAccounts))').value = @Model.ServiceAccountsCount;
+        } else {
+            document.getElementById('@(nameof(Model.SmServiceAccounts))').value = baseServiceAccounts;
+        }
+
+        // Clear autoscale values (no defaults)
+        document.getElementById('@(nameof(Model.MaxAutoscaleSmSeats))').value = '';
+        document.getElementById('@(nameof(Model.MaxAutoscaleSmServiceAccounts))').value = '';
+    }
+
+    function clearSecretsManagerConfiguration() {
+        document.getElementById('@(nameof(Model.SmSeats))').value = '';
+        document.getElementById('@(nameof(Model.SmServiceAccounts))').value = '';
+        document.getElementById('@(nameof(Model.MaxAutoscaleSmSeats))').value = '';
+        document.getElementById('@(nameof(Model.MaxAutoscaleSmServiceAccounts))').value = '';
+    }
+
+    function getBaseServiceAccountValue(planType) {
+        switch(planType) {
+            case '@((byte)PlanType.Free)':
+                return 3;
```

Comment:
```suggestion
                return @Bit.Core.Utilities.StaticStore.GetSecretsManagerPlan(PlanType.Free).BaseServiceAccount;
```
This way we don't have to worry about updating this if the value ever changes. Same for the other plans.

Reply:
This turned into a long switch statement that basically just matched planType to baseServiceAccount for that plan, so I went 1 better and just started bringing the plan objects directly into JS so we can look them up as required. We could use this for the Feature checkboxes as well - moving away from hardcoded values and providing more dynamic behaviour here. Let me know what you think.